### PR TITLE
streamline landingpage

### PR DESCRIPTION
**Pull request by haraldschilly**
# Description

* change the link at the top left to stay on the landingpage
* emphasize teaching on jupyter page

# Testing Steps
pages build, not much to check. I hope language & spelling is ok.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
- [ ] Run eslint on new and edited files
- [ ] All new code is actually used.
- [ ] Non-obvious code has some sort of comments.

Front end:
- [ ] Restart at least one project and check its `~/.smc/local_hub/local_hub.log`
- [ ] Completely restart Webpack with `./w` in `/src`
- [ ] Completely restart the hub by killing and restarting `./start_hub.py` in `/src/dev/project`
- [ ] Screenshots if relevant.

### CODE REVIEW

File: src/webapp-lib/doc/jupyter-notebook.pug [hunk 1]
```diff
@@ -41,30 +41,32 @@ block content
...
+          #{NAME} manages everything for you!
+        p.
+          Beyond that, #{NAME} is #[strong made for teaching]!
         ul
-          li don't bother setting up a Python environment
-          li you no longer have to fight with installing all the libraries you need
-          li don't worry about Jupyter kernel configurations
-          li and let #{NAME} maintain everything for you
+          li.
+            A sophisticated #[a(href="./teaching.html") course management] keeps track of all notebooks of all students.
```
Comment (williamstein) [inline src/webapp-lib/doc/jupyter-notebook.pug]: A sophisticated course management **system** keeps...

**Comment by williamstein** [comment]
What's left?

**Comment by haraldschilly** [comment]
I've experimented with some ideas yesterday. This isn't finished yet.

**Comment by johnjeng** [comment]
@haraldschilly can we get screenshots of the work so far for input?

**Comment by haraldschilly** [comment]
the artist is shy while the creative process is still ongoing :smile:

**Comment by haraldschilly** [comment]
I set this to needs review. I've made many of the changes we discussed yesterday, but not all of them. I wasn't happy with how they turned out. I also tweaked the text even more. As long as there aren't any serious typos, this should be ready to release. Later I'll look through all the other pages – they could need some face lifting as well – but not as part of this PR

**Pull request merged**